stan/math/fwd/meta/operands_and_partials.hpp

```diff
@@ -198,9 +198,9 @@ class ops_partials_edge<Dx, std::vector<std::vector<fvar<Dx>>>> {
   using partial_t = std::vector<Dx>;
   std::vector<partial_t> partials_vec_;
   explicit ops_partials_edge(const Op& ops)
-      : partials_vec_(length(ops)), operands_(ops) {
-    for (size_t i = 0; i < length(ops); ++i) {
-      partials_vec_[i] = partial_t(length(ops[i]), 0.0);
+      : partials_vec_(stan::math::size(ops)), operands_(ops) {
+    for (size_t i = 0; i < stan::math::size(ops); ++i) {
+      partials_vec_[i] = partial_t(stan::math::size(ops[i]), 0.0);
```

Comment:
This is because operand_and_partials has its own [size() function](https://github.com/stan-dev/math/blob/develop/stan/math/prim/meta/operands_and_partials.hpp#L151) and the compiler kept throwing errors.

Reply:
Cool, I had missed that.